### PR TITLE
Status synchro

### PR DESCRIPTION
**Pull request by Neckow**

### CODE REVIEW

File: RunningOrder/Shared/MainView.swift [hunk 1]
```diff
@@ -131,6 +131,6 @@ struct MainView_Previews: PreviewProvider {
     static var previews: some View {
         MainView()
             .environmentObject(SpaceManager.preview)
-            .environmentObject(AppStateManager())
+            .environmentObject(AppStateManager(changesService: CloudKitChangesService(container: CloudKitContainer.shared)))
```
Comment (zarghol) [inline RunningOrder/Shared/MainView.swift]: please make a preview such as SpaceManager above

File: RunningOrder/Sprint/Managers/SprintManager.swift [hunk 2]
```diff
@@ -15,13 +15,15 @@ final class SprintManager: ObservableObject {
     @Published var sprints: [Sprint] = []
 
     @Stored(fileName: "sprints.json", directory: .applicationSupportDirectory) private var storedSprints: [Sprint]?
+    @Published var isLoading = false
```
Comment (zarghol) [inline RunningOrder/Sprint/Managers/SprintManager.swift]: please remove this. as it was your previous implementation, dead code :)

**Review by zarghol — APPROVED** [review]

File: RunningOrder/Sprint/Managers/SprintManager.swift (hunk 2 above)
Comment (Neckow) [reply RunningOrder/Sprint/Managers/SprintManager.swift]: you dead

Comment (zarghol) [reply RunningOrder/Sprint/Managers/SprintManager.swift]: ☠️

**Review by ghiitais — APPROVED** [review]

**Pull request merged**